Duo_S/NuttX/README_zh.md

```diff
@@ -77,11 +75,11 @@ git clone https://github.com/apache/nuttx-apps apps
 
 Nuttx 需要使用 xPack 工具链进行编译，首先获取工具链，从 Github 下载，自行改变版本以匹配你的系统：
 ```bash
-wget https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack/releases/download/v11.5.0-1/xpack-riscv-none-elf-gcc-13.3.0-1-linux-x64.tar.gz
-tar -xvzf xpack-riscv-none-elf-gcc-13.3.0-1-linux-x64.tar.gz
+wget https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack/releases/download/v14.2.0-2/xpack-riscv-none-elf-gcc-14.2.0-2-linux-x64.tar.gz
+tar -xvzf xpack-riscv-none-elf-gcc-14.2.0-2-linux-x64.tar.gz
 ```
 
-将其添加到你的 PATH 中：
+/*建议将tar解压到/opt*/并将其添加到你的 PATH 中：
```

Comment:
/* */
What's this?
We don't write quotes like this in Markdown.
Remove it if you don't want it.

Reply:
Some stupid habits